src/pocket/PocketALBApplication.ts

```diff
@@ -737,103 +716,86 @@ export class PocketALBApplication extends Resource {
     const evaluationPeriods = {
       http5xxError: alarmsConfig?.http5xxError?.evaluationPeriods ?? 5,
       httpLatency: alarmsConfig?.httpLatency?.evaluationPeriods ?? 1,
-      httpRequestCount: alarmsConfig?.httpRequestCount?.evaluationPeriods ?? 1,
+    };
+    const http5xxAlarm: CloudwatchMetricAlarmConfig = {
```

Comment:
not to scope creep, but there is another breaking change ticket open for this repo. i don't know how much we care about releasing consecutive/multiple breaking changes updates.

the ticket is to rename `http5xxError` to a name that clearly communicates it's a percentage based error (and not a count based error): https://getpocket.atlassian.net/browse/BACK-1054

Reply:
@jpetto I can add that here